asciidoctor-revealjs.gemspec

```diff
@@ -29,4 +29,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'thread_safe', '~> 0.3.5'
 
   s.add_development_dependency 'rake', '~> 10.4.2'
+  s.add_development_dependency 'asciidoctor-doctest', '~> 2.0.0.beta.4'
```

Comment:
I’d be better to fix this version (`= 2.0.0.beta.4`), ’cause it’s a beta and there _may_ come some incompatible changes.